src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -67,7 +69,11 @@ export type DetailsViewProps = EntityDetailsListSharedProps & {
 /**
  * Describes the shape of the data passed to the BaseTable
  */
-export type DetailsViewRowData = (EntityHeader | ProjectHeader | Hit) & {
+export type EntityFinderTableViewRowData = (
```

Comment:
Slowly changing "DetailsView" -> "EntityFinderTableView". Didn't go all-in because I wanted this PR to have a readable diff